### ISSUE TITLE
Change the autostart question.

### ISSUE THREAD

**Issue by bb30994** [FoldingAtHome/fah-issues#1476]
When FAHClient is installed it asks if you want to autostart FAHClient when Windows is booted.  There's an easier way.  Simply change the question to "Do you want FAHClient to be started PAUSED?"  and reconfigure default for the option PAUSE-ON-START to be TRUE.  

Then they would be able to alter that choice later without reinstalling the client.  Then it also becomes a simple matter of manually doing an unpause to an already-running FAHClient daemon when they're ready to start it rather than finding the shortcut to manually start the client.  

In other words, set the existing pause-on-start option to true rather than the present default of false.

It seems that most people who reply YES to the existing question simply object to the default of the immediate onset of the processing FAHCore* (which slows up the Windows startup).

Treat FAHClient more like the daemon that it actually is.

**Comment by shorttack** [comment]
@bb Brilliant! Up voted: high priority.

**Comment by shorttack** [comment]
> the default of the immediate onset of the processing FAHCore* (which slows up the Windows startup).

A new FAHClient expert option would be PAUSE = <seconds>, [default=1] pausing start for N seconds to allow OS to get its startup housekeeping completed.

**Comment by PantherX** [comment]
I do like the delayed approach a lot more than the pause-on-start. Maybe it can be:
1. Would you like to automatically start folding upon Windows start (delayed pause)
2. Would you like to manually start folding upon Windows start (pause-on-start=true)

**Comment by TunaErtemalp** [comment]
> I do like the delayed approach a lot more than the pause-on-start. Maybe it can be:
> 
> 1. Would you like to automatically start folding upon Windows start (delayed pause)
> 2. Would you like to manually start folding upon Windows start (pause-on-start=true)

Just for the record, BOINC has a <start_delay> variable in their master cc_config.xml that allows the user set this delay in seconds. Something similar might be useful. The lack of such a setting was very obvious to me when I recently switched from full on BOINC to FAH.
